### PR TITLE
Editorial updates

### LINKED ISSUE #38: [Abstract] Not sure of the intended meaning of the following sentence...
@mwherman2000 moved from CCG (https://github.com/w3c-ccg/did-spec/issues/211)

### PR DESCRIPTION
**Pull request by rhiaro**
Solves most of #48, and should close #38 and #40.

***
<a href="https://pr-preview.s3.amazonaws.com/rhiaro/did-spec/pull/86.html" title="Last updated on Oct 26, 2019, 6:39 PM UTC (ee4c5e8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-spec/86/99a0873...rhiaro:ee4c5e8.html" title="Last updated on Oct 26, 2019, 6:39 PM UTC (ee4c5e8)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -139,17 +139,17 @@
...
 material, verification methods, and/or <a>service endpoints</a>. These provide
 a set of mechanisms which enable a <a>DID controller</a> to prove control of the
 <a>DID</a>. <a>Service endpoints</a> enable trusted interactions with the <a>DID
 subject</a>.
     </p>
     <p>
-This document specifies a common data model, format, and operations that
-all <a>DIDs</a> support.
+This document specifies a common data model, a URL format, and a set of operations for
+<a>DIDs</a> and related components.
```
Comment (TallTed) [inline index.html]:
```suggestion
<a>DIDs</a>, and related components.
```

Comment (peacekeeper) [inline index.html]: "DIDs and related components" doesn't make much sense to me; what are those "related components"? Syntax components like DID path or DID query? Or architectural components like DID registry and DID resolver?

**Comment by msporny** [comment]
@rhiaro can you make the changes please, I don't have commit rights to your branch.

@iherman, @brentzundel, @burnburn -- can we give @rhiaro and @grantnoble permission to open/edit PRs in the repo... it's going to be difficult for the editors to coordinate with them w/o this (and I expect a fair number of PRs from both of them over the next two years).

File: index.html (hunk 1 above)
Comment (rhiaro) [reply index.html]: I meant like DID documents and DID methods. I can change it to that.

**Comment by msporny** [comment]
Editorial, multiple positive reviews, comments addressed, merging.

**Comment by iherman** [comment]
@msporny, although this is moot as far as this repo is concerned, I have made some changes and @rhiaro, as well as @grantnoble have now the right to create a PR in this repo. However, only the editors should have 'merge' rights.

At least that is the intention... see if it works next time, @rhiaro !

**Comment by msporny** [comment]
> I have made some changes and @rhiaro, as well as @grantnoble have now the right to create a PR in this repo.

Great, thank you @iherman! :)

**Pull request merged**